Majasdarbs/03-majasdarbs/Gvido-CV-CSS/gvidozarins.html

```diff
@@ -0,0 +1,132 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Gvido Zarins | FE developer</title>
+    <link rel="stylesheet" href="styles.css">
```

Comment:
css files can be stored in a separate css folder. then import them like "css/styles.css"